src/legionpe/theta/utils/ReportErrorTask.php

```diff
@@ -38,19 +38,27 @@ public function onRun(){
 		if(microtime(true) - self::$last0 < 60){
 			return;
 		}
-		self::log("ping-all !!! PEMapModder ping!");
```

Comment:
Well xD This was supposed to notify PEMapModder on IRC, but this won't happen anymore ;)